### PR TITLE
bug fix: typo

### PR DESCRIPTION
**Pull request by JohnGavin**
eu option was not included
au option was included twice

### CODE REVIEW

**Comment by JohnGavin** [comment]
added an enhancement/bug fix (commit: cbb80bef48e1b4d9081b6faf6a8ae485499a7d54)
'active' column might be expected to be logical as it only contains TRUE/FALSE (and not of type character).

File: R/get-odds.R [hunk 1]
```diff
@@ -108,7 +108,7 @@ get_odds_raw <- function(sport = "upcoming",
 
     # Throw errors if query parameters aren't in the allowable
     # values for the API
-    region <- match.arg(region, c("us", "au", "uk", "au"))
+    region <- match.arg(region, c("us", "eu", "uk", "au"))
```
Comment (d-edison) [inline R/get-odds.R:111]: Good catch!

File: R/get-sports.R [hunk 2]
```diff
@@ -51,6 +51,7 @@ get_sports <- function(api_key = Sys.getenv("THEODDS_API_KEY")) {
             names_from = "name",
             values_from = "value"
         ) %>%
-        dplyr::bind_rows()
-
+        dplyr::bind_rows() %>%
+        # to avoid a character vector of TRUE/FALSE strings.
+        mutate(active = active %>% as.logical())
```
Comment (d-edison) [inline R/get-sports.R]: Yes, this should be logical, as should `has_outrights`. Could you remove that comment and switch this to 

```R
dplyr::mutate(
    dplyr::across(c(active, has_outrights), as.logical)
)
```

**Review by d-edison — CHANGES_REQUESTED** [review]
Appreciate the PR! Some small suggested changes

**Pull request merged**